relay/nakama/main.go

```diff
@@ -42,6 +44,15 @@ func InitModule(
 ) error {
 	utils.DebugEnabled = getDebugModeFromEnvironment()
 
+	// don't defer shutdown here, as it would shutdown otel immediately as it was initialized. we
+	// don't need to handle shutdown as nakama is supposed to be a long running process and is only
+	// stopped if the container itself is stopped (to be replaced by a new one).
```

Comment:
graceful termination by checking context.Done or OS signals doesn't work for some reason. since stopping nakama is almost always done in a container restart, it's probably safe to ignore this.